pandas/plotting/_core.py

```diff
@@ -1980,50 +1980,160 @@ def plot_series(data, kind='line', ax=None,                    # Series unique
 
 
 _shared_docs['boxplot'] = """
-    Make a box plot from DataFrame column optionally grouped by some columns or
-    other inputs
+    Make a box plot from DataFrame columns.
+
+    Make a box-and-whisker plot from DataFrame columns optionally grouped
+    by some other columns. A box plot is a method for graphically depicting
+    groups of numerical data through their quartiles.
+    The box extends from the Q1 to Q3 quartile values of the data,
+    with a line at the median (Q2).The whiskers extend from the edges
+    of box to show the range of the data. The position of the whiskers
+    is set by default to 1.5*IQR (IQR = Q3 - Q1) from the edges of the box.
```

Comment:
I'd use backticks for the formula, and use spaces as in Python, so \`1.5 * IQR (IQR = Q3 - Q1)\`. I think the render in html should look more clear.

Reply:
Backticks as code or cursive?

Reply:
Good point, not sure if it's code. It's your call, I think having a different format from the rest would help understand. May be \`\`IQR * 1.5\`\` where \`\`IQR = Q3 - Q1\`\`? ;) That seems like kind of Python propoer code. :)

But whatever you like, it was just an idea that I think would make it a bit easier to read.